util/data_generators.py

```diff
@@ -45,31 +45,47 @@ def generate_entry(conn, date: datetime, plant_id: str, location_center=(54.39,
     insert_processed_entry(conn, processed_entry)
 
 
+def insert_data(totals_knotweed, totals_rose, conn):
+    days = list(range(0, 360))
```

Comment:
I don't think this needs to be a list - in fact, can you define inline in the below list comp?

Reply:
oops, artefact from earlier version. Fixed ty :)